docs/build/iota-sdk/1.0.0/docs/how-tos/alias/destroy.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Destroy Alias Output
+description: 'How to programmatically destroy an alias output'
+image: /img/logo/iota_mark_light.png
+keywords:
+  - how to
+  - destroy
+  - alias
+  - output
+  - nodejs
+  - python
+  - rust
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You can destroy an alias output by ID. If the alias still owns any outputs when you try to destroy it, you will get an error.
```

Comment:
This is not correct, whether an alias output owns other outputs or not does matter when you are destroying one. There is nothing in place from a protocol level that would prevent you from burning/deleting an alias output of which its address are still present in other addresses. Likewise, you could also theoretically create outputs which are owned by an Alias address which does not even exist.

Can you maybe point out how you came to this conclusion?